libpromises/mod_common.c

```diff
@@ -262,6 +262,7 @@ const ConstraintSyntax CFG_CONTROLBODY[COMMON_CONTROL_MAX + 1] =
     ConstraintSyntaxNewReal("bwlimit", CF_VALRANGE, "Limit outgoing protocol bandwidth in Bytes per second", SYNTAX_STATUS_NORMAL),
     ConstraintSyntaxNewBool("cache_system_functions", "Cache the result of system functions. Default value: true", SYNTAX_STATUS_NORMAL),
     ConstraintSyntaxNewOption("protocol_version", "0,undefined,1,classic,2,latest", "CFEngine protocol version to use when connecting to the server. Default: classic", SYNTAX_STATUS_NORMAL),
+    ConstraintSyntaxNewOption("tls_ciphers", ".*", "List of acceptable ciphers in TLS connections that are initiated. For syntax help see man page for \"openssl ciphers\"", SYNTAX_STATUS_NORMAL),
```

Comment:
Can we set this to a predefined list to improve syntax checking, or does it need to be freeform in case OpenSSL is upgraded separately?